Userland/Libraries/LibWeb/HTML/HTMLImageElement.cpp

```diff
@@ -932,6 +932,32 @@ void HTMLImageElement::restart_the_animation()
     }
 }
 
+static bool is_supported_image_type(String const& type)
+{
+    if (type.is_empty())
+        return true;
+    if (!type.starts_with_bytes("image/"sv, CaseSensitivity::CaseSensitive))
+        return false;
+    // FIXME: These should be derived from ImageDecoder
+    if (type.equals_ignoring_ascii_case("image/bmp"sv)
```

Comment:
Why is this checking `image/` case-sensitively, but the whole mimetype insensitively?

From the spec I think both should be case-insensitive:

> All media type values, subtype values, and parameter names as defined are case-insensitive.
> - https://www.rfc-editor.org/rfc/rfc2045#section-2

Reply:
Ah that's my bad I was experimenting and forgot to change this back.